manifests/addons/app-autoscaler.yml

```diff
@@ -0,0 +1,435 @@
+releases:
+- name: app-autoscaler
+  url: https://app-autoscaler-boshrelease.s3.amazonaws.com/app-autoscaler.tgz
+  sha1: 7be27f150828f3f09499c72ef18dce4b4f5bd18d
+  version: 0+dev.4
+- name: consul
+  version: latest
+- name: routing
+  url: https://bosh.io/d/github.com/cloudfoundry-incubator/cf-routing-release?v=0.162.0
+  sha1: f17cf09d2414f5f486d18bbd57b17fd48fb69773
+  version: '0.162.0'
```

Comment:
We should probably not need to list the routing or consul releases explicitly, since those are core.